installing/installing_aws/installing-aws-outposts.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-aws-outposts"]
+= Installing a cluster on AWS with compute nodes on AWS Outposts
+include::_attributes/common-attributes.adoc[]
+:context: installing-aws-outposts
+
+toc::[]
+
+In {product-title} version 4.14, you could install a cluster on
+Amazon Web Services (AWS) with compute nodes running in AWS Outposts as a Technology Preview.
+As of {product-title} version {product-version}, this installation method is no longer supported.
+Instead, you can
```

Comment:
Totally up to you but if the alternative is at all important to highlight, making it its own paragraph makes it easier to see and makes the above paragraph smaller/more scannable. But this is bikeshedding so feel free to ignore.
```suggestion

Instead, you can
```